fantasy-docs.md

```diff
@@ -362,6 +393,37 @@ If your function accepts 0 parameters, then it can be visualized easily. But how
 
 This is a really powerful way to let consumers customize the goods you've designed before buying or manufacturing them. For example, you might put a design for a cool 3D printed office chair on thingiverse.com which has a function `main(name: Text)`. This function describes a chair with the given name embossed into the back. When a consumer wants to 3D print it, the 3D printing service will let them input their desired name, view the chair with that name embossed, and then order it.
 
+## Tests
+Functions are marked as tests using the `#[test]` attribute. They're run via the KittyCAD CLI or by the test runner built into the KittyCAD modeling app.
```

Comment:
I would specify that they should be placed on the line above a function for people who aren't familiar with what attributes are. Something like:
> Functions are marked as tests by placing the `#[test]` attribute on the line above them.